src/plugins/preauth/pkinit/pkinit_srv.c

```diff
@@ -1489,3 +1567,284 @@ kdcpreauth_pkinit_initvt(krb5_context context, int maj_ver, int min_ver,
     vt->return_padata = pkinit_server_return_padata;
     return 0;
 }
+
+/*
+ * Run the received, verified certificate through the certauth plugin interface,
+ * to verify that client can use the certificate based on the extension policy
+ * defined by modules.
+ */
+static krb5_error_code
+server_authorize_cert(krb5_context context, certauth_handle *certauth_modules,
```

Comment:
Do we need the server_ prefix on this name?  We're in pkinit_srv.c and this is a static function.